main.js

```diff
@@ -51,6 +44,18 @@ function encodeTweetForURI(tweetText) {
   return encodedTweet;
 }
 
+async function fetchQuotes() {
```

Comment:
how do you like async? I personally think it makes the reading flow nicer. Especially in combination of multiple promises (e.g. fetch-then-json)

Reply:
Yeah I like it a lot better than the Fetch API too for the same reason. As long as browser support isn't too much of an issue, I'd prefer to keep it.

Reply:
yeah. i think for most intents and purposes browser support is fine. let me know when you have more to review or when you want to have a chat again. merry xmas again!